content/applications/finance/accounting/bank/feeds/bank_statements.rst

```diff
@@ -6,67 +6,58 @@ Importing your bank statements in Odoo Accounting allows you to keep track of th
 movements that occur on your bank accounts and reconcile them with the transactions recorded in your
 accounting.
 
-We recommend you use bank synchronization for more efficiency. Please read the related
-documentation: :doc:`bank_synchronization`.
+We recommend using :doc:`bank_synchronization` for more efficiency. However, if you do not want to
+use **bank synchronization** or if your bank is not a supported institution, other options exist:
 
-However, if you don't want to use bank synchronization or if your bank is not a supported
-institution, you still have other options:
-
-#. Import the bank statement files delivered by your bank
-#. Register the bank statements manually
+- Import the bank statement files delivered by your bank;
+- Register the bank statements manually.
 
 Import bank statements files
 ============================
 
 Odoo supports multiple file formats to import bank statements:
 
-- SEPA recommended Cash Management format (CAMT.053)
-- Comma-separated values (.CSV)
-- Open Financial Exchange (.OFX)
-- Quicken Interchange Format (.QIF)
-- Belgium: Coded Statement of Account (.CODA)
+- SEPA recommended Cash Management format (CAMT.053);
+- Comma-separated values (.CSV);
+- Open Financial Exchange (.OFX);
+- Quicken Interchange Format (.QIF);
+- Belgium: Coded Statement of Account (.CODA).
 
-To import them, go to :menuselection:`Accounting --> Overview --> Bank`, click on *Import
-Statements*, or on the three dots, and then on *Import Statement*.
+To import a file, go to :menuselection:`Accounting Dashboard --> Overview --> Bank`, click on
+:guilabel:`Import Statements`, or click on the ⋮ button and then on :guilabel:`Import Statement`.
 
-.. image:: bank_statements/bank-statements-01.png
+.. image:: bank_statements/bank-overview.png
    :align: center
    :alt: Import a bank statement file in Odoo Accounting
 
-Next, select the file you want to import and click on *Import*.
+Next, select the file you want to import and click on :guilabel:`Import`.
 
-Odoo opens an **import widget** to help you set the **Formatting Options** and **map** the
-different columns you want to import.
+Odoo opens an **import widget** to help you set the :guilabel:`Formatting Options` and **map** the
+different columns you want to import. You also have the possibility to :guilabel:`test` the bank
+statement file before uploading it to your database.
 
-.. image:: bank_statements/bank-statements-02.png
+.. image:: bank_statements/import-bank-statement.png
    :align: center
    :alt: Register bank statements manually in Odoo Accounting
 
 .. note::
-   Quicken Interchange Format (.QIF) is an older file format that is no longer supported since 2005. 
-   If possible, prefer OFX files over QIF.
+   **Quicken Interchange Format (.QIF)** is no longer supported since 2005.
```

Comment:
```suggestion
   **Quicken Interchange Format (.QIF)** is no longer supported since 2005.
```
You can import QIF files but they are no longer supported? It's a bit confusing. I would either: remove that note or be more specific.

Reply:
I think what is happening here is that (.QIF) is no longer supported by whoever made it, but Odoo can still read the file format. I'll just remove the note altogether anyways